apps/web/src/pages/integrations/components/SetupFrameworkHeader.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from '@emotion/styled';
+import { Group, Stack, Title, UnstyledButton, useMantineColorScheme } from '@mantine/core';
+import { colors } from '@novu/notification-center';
+import * as capitalize from 'lodash.capitalize';
+import { shadows } from '../../../design-system';
+import {
+  AngularGradient,
+  ArrowLeft,
+  Close,
+  CodeGradient,
+  JsGradient,
+  ReactGradient,
+  VueGradient,
+} from '../../../design-system/icons';
+
+const Icon = ({ framework }: { framework: string }) => {
+  switch (framework) {
+    case 'react':
```

Comment:
Me and my typical suggestion of having an enum for the frameworks keys instead of hardcoded strings. 😅